README.md

```diff
@@ -33,15 +35,40 @@ More concretely, our goals are:
 
 ## Status
 
-The project is not ready for use. We hope to have full sync implemented sometime in Q1 2023, followed by optimizations. In the meantime, we're working on making sure every crate of the repository is well documented, abstracted and tested.
+The project is **not ready for production use**.
+
+We hope to have full sync implemented sometime in Q1 2023, followed by optimizations. In the meantime, we're working on making sure every crate of the repository is well documented, abstracted and tested.
```

Comment:
I think this is no longer up to date, since Q1 has passed and we also are able to fully sync